src/app.scss

```diff
@@ -78,6 +78,11 @@
 
 button.pf-m-secondary.breadcrumb-edit-button {
     margin-inline-end: var(--pf-v5-global--spacer--sm);
+
+    // PatternFly doesn't expect icons in buttons in breadcrumbs, so we need to adjust for that
```

Comment:
This comment seems wrong? This is the edit path button.

Reply:
The edit path button is a button with an icon and it's nested under the breadcrumb component... that's _exactly_ what this comment is talking about.

I'm pretty sure the breadcrumb component is supposed to contain breadcrumbs and nothing else. See my comment above @ https://github.com/cockpit-project/cockpit-files/pull/542#issuecomment-2179037558

I added some workaround CSS meanwhile, which is what this comment is talking about.

Reply:
(The thing is, it also affects the bookmark dropdown, which also is in the breadcrumb widget, which also should probably not be. And to fix one, I needed to fix both and refactor the CSS a bit, which is why it's in this PR.)